.github/workflows/test-k8s-lib-injection.yaml

```diff
@@ -39,8 +39,10 @@ jobs:
         run: |
           npm pack
           cp dd-trace-*.tgz k8s/auto-inst/dd-trace.tgz
-          docker build k8s/auto-inst --build-arg npm_pkg=./dd-trace.tgz -t ghcr.io/datadog/dd-trace-js/dd-trace-js-init:${GITHUB_SHA}
-          docker push ghcr.io/datadog/dd-trace-js/dd-trace-js-init:${GITHUB_SHA}
+          docker buildx create --name lib-injection
+          docker buildx use lib-injection
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes # https://stackoverflow.com/questions/72167570/docker-buildx-nodejs-fail
+          docker buildx build --platform=linux/amd64,linux/arm/v7,linux/arm64/v8 --build-arg npm_pkg=./dd-trace.tgz -t ghcr.io/datadog/dd-trace-js/dd-lib-js-init:${GITHUB_SHA} --push k8s/auto-inst
```

Comment:
Should this test use the exact same action as the release process to ensure that we're testing what we ship?

Reply:
It does now! The only difference is whether or not the `--push` arg is provided.

Would you rather I encapsulate this in a script so that it's less likely to diverge?

Reply:
Err, no, now I see what you're saying. Let me go ahead and do that.